<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>651</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SmartCalc v2.0</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="OutputLine">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>550</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	border: 1px solid gray;
	background-color: white;
	color: black;
}</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="margin">
     <number>10</number>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::LinksAccessibleByKeyboard|Qt::LinksAccessibleByMouse|Qt::TextBrowserInteraction|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_enter">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>0</y>
      <width>50</width>
      <height>100</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: 1px solid gray;
	background-color: silver;
	color: black;
}

QPushButton::pressed {
	border: 1px solid black;
	background-color: grey;
	color: black;
}</string>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="InputExpressionLine">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>550</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
	border: 1px solid gray;
	background-color: white;
	color: black;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="InputVarLine">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>100</y>
      <width>160</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
	border: 1px solid gray;
	background-color: white;
	color: black;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="x_value_calc">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>100</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	border: 1px solid gray;
	background-color: white;
	color: black;
}</string>
    </property>
    <property name="text">
     <string>x ↑</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="margin">
     <number>10</number>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_plot">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>100</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: 1px solid gray;
	background-color: silver;
	color: black;
}

QPushButton::pressed {
	border: 1px solid black;
	background-color: grey;
	color: black;
}</string>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QCustomPlot" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>151</y>
      <width>600</width>
      <height>500</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="InputAxisRangeXmin">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>100</y>
      <width>60</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
	border: 1px solid gray;
	background-color: white;
	color: black;
}</string>
    </property>
    <property name="text">
     <string>-10</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="InputAxisRangeXmax">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>100</y>
      <width>60</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
	border: 1px solid gray;
	background-color: white;
	color: black;
}</string>
    </property>
    <property name="text">
     <string>10</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="x_value_graf">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>100</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	border: 1px solid gray;
	background-color: white;
	color: black;
}</string>
    </property>
    <property name="text">
     <string>x ↓</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="margin">
     <number>10</number>
    </property>
   </widget>
   <widget class="QLabel" name="y_value_graf">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>100</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	border: 1px solid gray;
	background-color: white;
	color: black;
}</string>
    </property>
    <property name="text">
     <string>y ↓</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="margin">
     <number>10</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="InputAxisRangeYmin">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>100</y>
      <width>60</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
	border: 1px solid gray;
	background-color: white;
	color: black;
}</string>
    </property>
    <property name="text">
     <string>-10</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="InputAxisRangeYmax">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>100</y>
      <width>60</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
	border: 1px solid gray;
	background-color: white;
	color: black;
}</string>
    </property>
    <property name="text">
     <string>10</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
